go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/go-vela/server
 
+replace github.com/go-vela/compiler => github.com/JordanSussman/compiler v0.1.3-0.20210709181443-362fa898a72c
```

Comment:
This will be removed once https://github.com/go-vela/compiler/pull/199 has been merged.